sycl/test-e2e/Basic/buffer/buffer.cpp

```diff
@@ -656,17 +656,15 @@ int main() {
     sycl::buffer<char, 1> Buf_2(Size / 2);
 
     {
-      auto AccA = Buf_1.get_access<sycl::access::mode::read_write>(Size / 2);
-      auto AccB = Buf_2.get_access<sycl::access::mode::read_write>(Size / 2);
+      auto AccA = Buf_1.get_host_access(Size / 2, sycl::read_write);
+      auto AccB = Buf_2.get_host_access(Size / 2, sycl::read_write);
```

Comment:
Nit; AFAIK, though this is completely valid, the preferred way of constructing an accessor is using the deduction guides, i.e.

```suggestion
      sycl::host_accessor AccA{Buf_1, Size / 2, sycl::read_write};
      sycl::host_accessor AccB{Buf_2, Size / 2, sycl::read_write};
```